website/pages/docs/deployment/github-actions.md

```diff
@@ -0,0 +1,144 @@
+  path: cloudquery/aws
+  version: 'v4.1.0' # latest version of aws plugin
+  destinations: ['postgresql']
+---
+kind: destination
+spec:
+  name: 'postgresql'
+  path: cloudquery/postgresql
+  version: 'v1.6.0' # latest version of postgresql plugin
+  spec:
+    connection_string: ${CQ_DSN} # The CQ_DSN environment variable will be set by GitHub Action workflow
+```
+
+> For more configuration options, [visit our docs](/docs/reference/source-spec)
+
+## Creating the GitHub Action workflow
+
+First we'll need [to create a GitHub secret](https://docs.github.com/en/actions/security-guides/encrypted-secrets#creating-encrypted-secrets-for-a-repository) with the name `CQ_DSN` and the value of the connection string to your PostgreSQL database.
+
+Create a workflow file under `.github/workflows/cloudquery.yml` with the following content, and fill in `<role-arn>` and `<region>` according to the role you created in the prerequisites.
+
+```yaml
+name: CloudQuery
+on:
+  schedule:
+    - cron: '0 3 * * *' # Run daily at 03:00 (3am)
+
+jobs:
+  cloudquery:
+    permissions:
+      id-token: write
+      contents: read
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3 # Checkout the code so we have access to the config file
+      - name: Configure AWS credentials # Setup AWS credentials (example)
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          role-to-assume: <role-arn> # based on the role you created in the prerequisites
+          aws-region: <region> # based on the region you created the role in
+      - uses: cloudquery/setup-cloudquery@v2
+        name: Setup CloudQuery
+      - name: Sync with CloudQuery
+        run: cloudquery sync cloudquery.yml --log-console
+        env:
+          CQ_DSN: ${{ secrets.CQ_DSN }} # Connection string to a PostgreSQL database
+```
+
+Once committed to the default branch of the repository, the above workflow will run daily at 3 a.m. and will sync the AWS source plugin with the PostgreSQL destination plugin.
+
+## Running CloudQuery in parallel to speed up sync time
+
+By default, CloudQuery extracts all supported resources, which can take a bit of time, depending on the number of resources you have in your AWS account.
+
+With the [GitHub Actions matrix configuration](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs), you can split the sync process into multiple jobs and run them in parallel.
+
+> In the example below, we'll split the sync process using regions, but you can split it by any other dimension, such as tables, accounts or any combination that makes sense for your use case.
+
+First, we'll need to create a new `cloudquery-regions.yml` configuration file under the root of the repository:
+
+```yaml
+kind: source
+spec:
+  name: 'aws-REGION_PLACEHOLDER' # when splitting configurations, we need to keep the names unique
+  path: cloudquery/aws
+  version: 'v4.1.0' # latest version of aws plugin
+  destinations: ['postgresql']
```

Comment:
needs to also be `postgresql-REGION_PLACEHOLDER` (otherwise 
<img width="1156" alt="image" src="https://user-images.githubusercontent.com/37939765/199687471-9f7ea1c0-781d-48f6-9290-cd7f4b4b616e.png">
)

Reply:
Nice catch